### PR TITLE
Update docs with paymentData

### PR DESCRIPTION
**Pull request by mariovalney**
Hi! Me again.

I was able to finish a transaction yeasterday and noticed the paymentData was missing the header field.
It's useful for the gateway I was using so I guess can help futher developers showing this field.

Thanks again

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -84,12 +84,16 @@ ApplePaySessionPolyfill.createPaymentToken = function (session) {
       },
       billingContact: ApplePaySessionPolyfill.createBillingContact()
     },
-    transactionIdentifier: '999999999999999',
+    transactionIdentifier: 'AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA'
```
Comment (martincostello) [inline README.md]:
```suggestion
    transactionIdentifier: 'AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA',
```

File: README.md [hunk 2]
```diff
@@ -84,12 +84,16 @@ ApplePaySessionPolyfill.createPaymentToken = function (session) {
...
...AAA9AAAaaAAAAaaAAaAA9aAaAAAAaAA9AAAaAAAAAaAAAAAaaaaaaAAAAAAaaaaaaAAAAAAaAAAAAaAaAAAaaaaaAaAaAAA9aaaAAa9aaaAaAA/AaA9AAA9AAaAa9a9Aa9AAAAa9aA/AAAaaaAaaaaaA9A9AaA/9AAa9A9AAA9AAAAAAaAAAAAAaAAAAaaAAA9AaaaAaAAAAaaAAAaaAA9aAaaA99aa9AaAAAaaAAaA99aA99aa9AaAA9aaAAaAAAaa9AaAAaaAA9AA9AaAaAAAAAa9aaAA9Aa9aaaaAAaaA+aa/9aAaAAAaAAAAAAAa9AAAAAAAA/AA9AA9AaAaAAAAaAAAaa9aAAA9aaaAaA9a9999/aaaAaAAaAA9AaAaAaAA9aAAAaaAaAAaa9aAA9Aa9aaaaaAAAaaAAaA99aA9AaaAaaaa9aaAAaAaAaA9AaAA9AA9AaAaAA/aAAAaAAAaAAAaaaaaaA99AaAaAAAAAAAAAAAaaaaaaAAAAAAaAaAAAaAaA9a9AAAAaAaAa9AaAAaaA/9Aaa9A99AaAAaaAaaA9Aa9a9aAaaAA+A9AAAa99aaa9AAAAa9AAaaAAAa9aa9aa9aAa9A9aaAA9AA9AaaAAAa9A9aa9aaaaaAAaaAAaAAaaAAAAAAAAAaAAAAAaAAAAaaAAaaaAaAAaAA9AAAaaaAAAaaAAaAAAaaAaaAAAaa99aAA99AAaaAAAaa99aA9AaAAAAAaAaAAAAA9AAAaaaAAAaaAAaA9AaaAaaaAAaaAaaaaAAaAAaa9AaaAaaAaAAAaAAAAaAAaAaaAaaAAaaAa9aAaAAAaAAAAAAAaAAAaaAAAAAAA9AAaAAAaaaaaaAAAAAAaAAAAAAaAAAAaaaaaaA9a9AAAAaAaAAAaAAaaaAAAaAAAaAAAaAAAa9AAAAAAAAAa9aAAA9AaAaAAAaAAAaAAaAAAaAAAa9AAAAAAAaAAaaAAAAAAAAAAAAAAAAAAAaAaAAAaAAaa9AAaAaAaAAAaAAaaaAAAaAAAAAAA9Aa9aAAAAA9AAaAaA9a9aA9AAaaaA9AaAaaaaA/9AAAAaAAAaAAA99AAAAAAaaAAAaAA9aaAAAaAaaAA9AAAaaaAAaAaAA9Aa9AAaAAaaaAaAaAaAAAaAA+a9A9AAaaaaaaaaA9a9aaaAAa+AAA9Aaa99aAAAAAAAAAA==",
```
Comment (martincostello) [inline README.md]: Do these need to have quite so much data in them? It introduces quite a big horizontal scrollbar.

Comment (mariovalney) [reply README.md]: Actually I'm not sure. haha
I just redacted keeping letters and numbers of the original.

But yes... I agree it's maybe too much.

**Review by martincostello — APPROVED** [review]

**Comment by mariovalney** [comment]
Done. Fixed the length of the datas.

**Pull request merged**